tagging/tagger_interface.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+
+class TaggerInterface:
+    """HooksInterface for all hooks common interface"""
+    @staticmethod
+    def tag_value(image):
+        raise NotImplementedError
+
+    @staticmethod
+    def tag_name():
+        raise NotImplementedError
```

Comment:
This is just to make logs look good, and I think it's nice to use classes for Taggers instead of pure functions - this will give an ability to add manifests in these taggers without a lot of changes.

Reply:
Decided to delete this and use `__name__` property - still looks good and much less boilerplate code.